ubuntu-ruby-fips/README.md

```diff
@@ -1,11 +1,12 @@
 # Ubuntu container image
 This container image includes Ubuntu version `20.04` which contains the following packages:
 
-* OpenSSL version `1.0.2zg`: built by SafeLogic to be FIPS compliant.
-* Ruby version `3.0.6`: compiled against the FIPS 140-2 compliant OpenSSL module.
-* Postgres client version `10-10.16`: compiled against the FIPS 140-2 compliant OpenSSL module.
-* Bundler version `2.2.33`.
- 
+* OpenSSL: built by SafeLogic to be FIPS compliant.
```

Comment:
I think this line is not accurate anymore since we compile the FIPS module ourselves, and use the OpenSSL package provided by the distribution, right?